runtime/protocol/http-test/jvm/src/aws/smithy/kotlin/runtime/httptest/TestWithLocalServer.kt

```diff
@@ -36,7 +33,7 @@ public abstract class TestWithLocalServer {
                 attempt++
                 try {
                     server.start()
-                    logger.info { "test server listening on: $testHost:$serverPort" }
+                    println("test server listening on: $testHost:$serverPort")
```

Comment:
should `println` be used here? I see you removed the log statement entirely in `OkHttpProxySelector`, should that also have a `println`?

Reply:
This is in a test, I'm less worried about it. The `OkHttpProxySelector` would need to use logs but it lacks a `CoroutineContext` to pull a logger from. I _think_ this is actually ok though because the [HttpEngineEventListener](https://github.com/awslabs/smithy-kotlin/blob/a250c3e3e3e54ef35990a1609fb380a91b70cf4b/runtime/protocol/http-client-engines/http-client-engine-okhttp/jvm/src/aws/smithy/kotlin/runtime/http/engine/okhttp/HttpEngineEventListener.kt#L53)  for the request is going to capture the same connect failure that was being logged.